internal/bed/sorting.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/maruel/natural"
 )
 
+// Note that only lowercase is used in this slice
+var humanChrOrder = []string{"1", "chr1", "2", "chr2", "3", "chr3", "4", "chr4", "5", "chr5", "6", "chr6", "7", "chr7", "8", "chr8", "9", "chr9", "10", "chr10", "11", "chr11", "12", "chr12", "13", "chr13", "14", "chr14", "15", "chr15", "16", "chr16", "17", "chr17", "18", "chr18", "19", "chr19", "20", "chr20", "21", "chr21", "x", "chrx", "y", "chry", "mt", "chrmt"}
```

Comment:
I think if the prefix `chr` is used, the mitochondrial genome is `chrM`. Only without prefix it is `MT`.

I was also thinking (just an idea, not sure if it is feasible) if natural sorting can be applied and by using a regex for the mitochondrial genome you could do the shift to the end of the main chromosomes.

Reply:
Thank you, I was not aware about chrM this!
I see now that both chrM and chrMT is used, so I'm adding both M and chrM to the list.